### ISSUE TITLE
The currency converter does not work 

### ISSUE THREAD

**Issue by ShivangiSingh17** [project-travel-mate/Travel-Mate#690]
### Description
The feature "currency converter" available under the ```Utilities``` section does not.

### Steps to reproduce
Utilities>Currency Converter
### What the current behaviour is?
The converter does not convert the value provided in the desired currency. Eg. If we convert Rs.1 to bitcoin it is showing value to be 00001375, 0.000001462 etc. and the actual value is much higher to about 764586.5
### What the behaviour should?
The converter should work.

### Would you like to work on this issue?
Yes

**Comment by Swati4star** [comment]
@ShivangiSingh17 Sure

**Comment by louelidrissi** [comment]
Is someone taking over this ?

**Comment by Swati4star** [comment]
No @lelmedda 
You want to take up this issue?

**Comment by nasreekar** [comment]
Can I work on this if this is not taken up yet? 
Also, are there any developer credentials which I can use. There seems to be an issue with creating a new account (throwing unable to send email error message)

**Comment by Swati4star** [comment]
Yes, you can. @nasreekar 
you can use these credentials:
email: testing123@gmail.com
password: Test@123

**Comment by nasreekar** [comment]
@Swati4star  Looks like the API is not working. 
Not receiving any value from the API call.

`Response{protocol=http/1.1, code=503, message=Service Unavailable, url=https://project-travel-mate.herokuapp.com/api/get-currency-conversion-rate/USD/INR}`

Can you check if there is any problem with the API at the backend ?

**Comment by Swati4star** [comment]
@nasreekar Yes, there seems to be some issue with this service. @prabhakar267 will look into this.

**Comment by nasreekar** [comment]
let me know once it is fixed so that I can pick it up.  :) 

Also, an _enhancement_ suggestion.
If there is an error with the converter, it is showing an animation but then clicking on the back button is dismissing the whole activity and navigating the user to utilities fragment. It would be great if the user is navigated to the currency converter activity instead of utilities so that the user doesn't have to do it again from the beginning.

**Comment by Swati4star** [comment]
Sure @nasreekar 
Also, can you open up another issue for the suggestion you mentioned? If you want, you can start working on that too.
